<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Line_Width</class>
 <widget class="QWidget" name="Line_Width">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>銅厚</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0,0" columnstretch="1,1">
     <item row="1" column="1">
      <widget class="QLabel" name="label_12">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>oz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="thickness_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="Mass_lineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="thickness_comboBox"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>銅箔重量</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>銅箔厚度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>參數設定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13" columnstretch="1,1">
     <item row="3" column="1">
      <widget class="QLabel" name="label_24">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>攝氏</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="temp_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_21">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電流</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>走線長度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="Current_lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="Length_lineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_16">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>單位</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="Length_comboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="Current_comboBox"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_22">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>溫升</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_23">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>單位</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>661</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Scientific_computing.qrc">:/Line_With.bmp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>總結</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,1,1,1,1,1,1" columnstretch="1,1,1,1">
     <item row="4" column="1">
      <widget class="QComboBox" name="Internal_comboBox"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_20">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>V</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="External_comboBox"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_17">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>走線電阻</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="Impedance_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="Internal_lineEdit"/>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_25">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>W</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="Impedance_comboBox"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_26">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>外層寬度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="VoltageDrop_lineEdit"/>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="VoltageDrop_comboBox"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>銅箔寬度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="External_lineEdit"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_27">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>內層寬度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_19">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>功耗</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_18">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>走線壓降</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QComboBox" name="Consumption_comboBox"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="Consumption_lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Scientific_computing.qrc">:/Line_With-power.bmp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Scientific_computing.qrc"/>
 </resources>
 <connections/>
</ui>
